### PR TITLE
Add Matlab wrapper to allow legacy to new multifit

### LINKED ISSUE #1234: Add wrapper for legacy multifit converter to call from matlab
C.f. #1221 #1226

### PR DESCRIPTION
**Pull request by oerc0122**
Add short wrapper script into `admin` to be able to call converter from Matlab

Fixes #1234

### CODE REVIEW

File: admin/mf_leg_to_new.m [hunk 1]
```diff
@@ -0,0 +1,28 @@
...
+% J. Wilkins 8-9-2023
+
+
+
+    pth = [horace_paths.root, pathsep, 'tools'];
+    if count(py.sys.path, pwd) == 0
+        insert(py.sys.path, int32(0), pth);
+    end
+
+    conv = py.mf_leg_to_new.convert_legacy_multifit(funcline);
```
Comment (mducle) [inline admin/mf_leg_to_new.m:26]: A nicer output might be:

```suggestion
    convstr = strip(char(py.mf_leg_to_new.convert_legacy_multifit(funcline)));
    if nargout < 1
        disp(convstr)
    else
        conv = convstr;
    end
```

**Review by mducle — APPROVED** [review]

**Review by mducle — CHANGES_REQUESTED** [review]
Actually I just realised that the `admin/` folder is not added to the Matlab path when you do `horace_init` (or `horace_on` on IDAaaS). So it would be best to put this file in another folder which is on the path, like `horace_core/utilities`

**Comment by abuts** [comment]
There is another `admin` folder, located under `horace_core` (or `herbert_core`) this folder is added to the path while top admin is intended for devalopment and installation scripts

**Review by abuts — CHANGES_REQUESTED** [review]
move it to `horace_core\admin` folder and than it looks fine.

**Review by mducle — APPROVED** [review]

File: horace_core/admin/mf_leg_to_new.m [hunk 2]
```diff
@@ -0,0 +1,33 @@
+function conv = mf_leg_to_new(funcline)
```
Comment (cmarooney-stfc) [inline horace_core/admin/mf_leg_to_new.m:1]:
```suggestion
function conv = mf_legacy_to_new(funcline)
```
Don't think missing out 3 chars is worth the uncertainty in meaning

**Review by cmarooney-stfc — CHANGES_REQUESTED** [review]

**Pull request merged**